templates/package.xml.template

```diff
@@ -24,13 +24,9 @@
    </stability>
    <license>Apache 2.0</license>
    <notes>
-  - Fixed some memory leaks #9559, #10996
-  - Disabled cares dependency from gRPC C Core #10940
-  - De-coupled protobuf dependency #11112
-  - Fixed extension reported version #10842
-  - Added config.w32 for Windows support #8161
-  - Fixed PHP distrib test after cc files were added #11193
-  - Fixed protoc plugin comment escape bug #11025
+  - Channel are now by default persistent #11878
```

Comment:
SG. This is for the `master` branch so we will still have time for the next feature intended for 1.7 so I can fix the release notes then.

I will do what you suggest when I work on the `v1.6.0` branch release notes. Thanks.